### PR TITLE
➕ Add Replit and GoLand badges

### PR DESCRIPTION
**Pull request by kohasummons**
🧪 Fulfill badge requests in #346
🔨 Fix broken emoji

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -508,13 +508,15 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
...
 | PhpStorm           | ![PhpStorm](https://img.shields.io/badge/phpstorm-143?style=for-the-badge&logo=phpstorm&logoColor=black&color=black&labelColor=darkorchid)        | `![PhpStorm](https://img.shields.io/badge/phpstorm-143?style=for-the-badge&logo=phpstorm&logoColor=black&color=black&labelColor=darkorchid)`        |
 | PyCharm            | ![PyCharm](https://img.shields.io/badge/pycharm-143?style=for-the-badge&logo=pycharm&logoColor=black&color=black&labelColor=green)                | `![PyCharm](https://img.shields.io/badge/pycharm-143?style=for-the-badge&logo=pycharm&logoColor=black&color=black&labelColor=green)`                |
+| Replit            | ![Replit](https://img.shields.io/badge/Replit-DD1200?style=for-the-badge&logo=Replit&logoColor=white)                | `![Replit](https://img.shields.io/badge/Replit-DD1200?style=for-the-badge&logo=Replit&logoColor=white)`                |
```
Comment (Ileriayo) [inline README.md:519]: Duplicate badge.

See [PR 145](https://github.com/Ileriayo/markdown-badges/pull/145)

File: README.md [hunk 2]
```diff
@@ -508,13 +508,15 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
...
 | Eclipse            | ![Eclipse](https://img.shields.io/badge/Eclipse-FE7A16.svg?style=for-the-badge&logo=Eclipse&logoColor=white)                                      | `![Eclipse](https://img.shields.io/badge/Eclipse-FE7A16.svg?style=for-the-badge&logo=Eclipse&logoColor=white)`                                      |
 | Emacs              | ![Emacs](https://img.shields.io/badge/Emacs-%237F5AB6.svg?&style=for-the-badge&logo=gnu-emacs&logoColor=white)                                    | `![Emacs](https://img.shields.io/badge/Emacs-%237F5AB6.svg?&style=for-the-badge&logo=gnu-emacs&logoColor=white)`                                    |
+| GoLand              | ![GoLand](https://img.shields.io/badge/GoLand-0f0f0f?&style=for-the-badge&logo=goland&logoColor=white)                                    | `![GoLand](https://img.shields.io/badge/GoLand-0f0f0f?&style=for-the-badge&logo=goland&logoColor=white)`                                    |
```
Comment (Ileriayo) [inline README.md:511]: 👍

**Review by Ileriayo — CHANGES_REQUESTED** [review]
Remove duplicate Repl badge.

File: README.md (hunk 1 above)
Comment (kohasummons) [reply README.md:519]: Okayy. On it.

**Comment by Ileriayo** [comment]
> Question: Should Repl be under the Education category or Editor Category?
> 
> @Ileriayo

Great question.
It should be under IDEs/Editors.

**Comment by kohasummons** [comment]
🔨 Removed Duplicate replit badge.

**Review by Ileriayo — APPROVED** [review]

**Comment by Ileriayo** [comment]
All good now.

**Comment by kohasummons** [comment]
Okay. 🎊

**Pull request merged**